src/common/utils.ts

```diff
@@ -445,7 +445,7 @@ export const generateSecrets = async (values: Record<string, unknown> = {}): Pro
   debug.info('Second round of templating')
   const secondTemplateRound = (await gucci(firstTemplateRound, gucciOutputAsTemplate, {
     asObject: true,
-  })) as Record<string, unknown>
+  })) as Record<string, any>
```

Comment:
I notice you replaced all occurrences of `any` with `unknown`. I think this is regression from a type safety perspective. I am by far a TS expert, but here was a useful SO article explaining the difference: https://stackoverflow.com/questions/51439843/unknown-vs-any.

Reply:
Unknown is is type safe.
> [..] unknown which is the type-safe counterpart of any. Anything is assignable to unknown, but unknown isn't assignable to anything but itself and any without a type assertion or a control flow based narrowing. Likewise, no operations are permitted on an unknown without first asserting or narrowing to a more specific type
https://stackoverflow.com/questions/51439843/unknown-vs-any

Reply:
fine then...lets keep typescript strictness for another discussion

Reply:
ok